core/arch/x86/instr.c

```diff
@@ -2027,6 +2027,184 @@ opnd_same_sizes_ok(opnd_size_t s1, opnd_size_t s2, bool is_reg)
     return (s1_default == s2_default);
 }
 
+opnd_size_t
+instr_get_input_size_from_opcode(int opcode, bool evexw)
```

Comment:
This function was declared in opnd.h, and implemented in instr.c?  I think it should be declared in decode_private.h and implemented in decode.c and named decode_.

Reply:
Done, see above.